src/explore-education-statistics-frontend/src/modules/table-tool/components/WizardStepFormActions.tsx

```diff
@@ -12,6 +12,7 @@ interface Props {
   stepNumber: InjectedWizardProps['stepNumber'];
   submitText?: string;
   submittingText?: string;
+  submitOnClick?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
```

Comment:
- Event handler naming convention should be 'onSomething', so would suggest changing this to `onSubmitClick`
- Can use `MouseEventHandler<HTMLButtonElement>` instead